_posts/2022-05-22-openmic-stamp-announcement.markdown

```diff
@@ -0,0 +1,14 @@
+---
+title: "Upcoming Open Mic - Ontological Foundation of Hazards and Risk in STAMP"
+categories: [Open Mic Announcement, Open Mic]
+---
+
+On friday 27th May at 10:30, another open mic session presented by Jana Ahmad will take place.
```

Comment:
Friday

Reply:
Also. I think each session should have a one or two sentence annotation that could be put in the announcement/archive post. So that the reader as at least some idea of the topic of the talk.

Reply:
First paragraph is excerpt visible in archive and also in post list (it may be setup for every occurrence). Everything in the first paragraph is an excerpt (maybe there is some limit on number of chars, IDK)

Reply:
Ok, but I don't see the excerpt in the page with open mic announcements.